pkgs/development/haskell-modules/configuration-common.nix

```
@@ -855,7 +855,13 @@ self: super: {
   swagger2 = if (pkgs.stdenv.hostPlatform.isAarch32 || pkgs.stdenv.hostPlatform.isAarch64) then dontHaddock (dontCheck super.swagger2) else super.swagger2;
 
   # hledger-lib requires the latest version of pretty-simple
-  hledger-lib = super.hledger-lib.override { pretty-simple = self.pretty-simple; };
+  hledger-lib = appendPatch
+    (super.hledger-lib.override { pretty-simple = self.pretty-simple; })
```

The old override was unnecessary and can be removed:

```suggestion
  hledger-lib = appendPatch super.hledger-lib
```

---

Fixed in a dedicated commit

---

I see you also did it and it already got merged in master so I removed my commit.